src/resources/editor/tools/vs-code.mjs

```diff
@@ -28077,10 +28082,8 @@ function postProcessAnnotation(parse) {
   if (parse.components.length === 1 && parse.start === parse.components[0].start && parse.end === parse.components[0].end) {
     return postProcessAnnotation(parse.components[0]);
   } else {
-    return {
-      ...parse,
-      components: parse.components.map(postProcessAnnotation)
-    };
+    parse.components = parse.components.map(postProcessAnnotation);
```

Comment:
@dragonstyle I'm not sure why in my change the code gen has now been refactored to prefer a non-mutable approach.

Reply:
I def can't explain it - those files are generated so perhaps something changed (e.g. a dependency) that resulted in them being generated a slightly different way...

Reply:
Thanks for reviewing.